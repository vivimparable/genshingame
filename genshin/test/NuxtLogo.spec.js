import { shallowMount } from '@vue/test-utils'
import mainPage from '@/pages/index.vue'

describe('MainPageGenshim', () => {
 let wrapper = shallowMount(mainPage)
 beforeEach(()=>{
  wrapper = shallowMount(mainPage)

})
      test('Snapshot', () => {
        expect(wrapper.element).toMatchSnapshot()
      })


      test('Debe haber un h1', () => {
        let header = wrapper.find('h1')
        expect(header).toBeTruthy()
      })

      test('Debe haber un boton', () => {
        let boton = wrapper.find('button')
        expect(boton).toBeTruthy() 
      })

      test('El boton debe devolver una ruta a pjchose',    () => {

       
       
        
        
      })
})
