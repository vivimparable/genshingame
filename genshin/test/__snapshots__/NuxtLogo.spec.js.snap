// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPageGenshim Snapshot 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="col"
    >
      <h1
        class="text-center"
      >
        Welcome to the genshin impact revision
      </h1>
    </div>
     
    <div
      class="col"
    >
      <button
        class="btn"
      >
        <h2>
           Start
        </h2>
      </button>
    </div>
  </div>
</div>
`;
